### ISSUE TITLE
Dynamic Forms field tokens can't be chained on

### ISSUE THREAD

**Issue by Piedone** [OrchardCMS/Orchard#7702]
Take a token like this to handle a form submission: `{FormSubmission.Field:Message}.` If you'd like to chain other tokens on this, like Text tokens (which are explicitly made chainable in `FormTokens`) you'd do this:  `{FormSubmission.Field:Message.Text.LineEncode}`.

However this won't work because `FormTokens` will think everything chained is part of the field name, i.e. the filed is called "Message.Text.LineEncode", and thus the token evaluation will just yield an empty string.

**Comment by HermesSbicego-Laser** [comment]
Similar issue was solved with #7356 in which we chain text in querystring and form tokens.

**Comment by Piedone** [comment]
Indeed. Not sure if we should have a generic solution or just decide that form field names can't have dots in them and splitting the string on that.

**Comment by HermesSbicego-Laser** [comment]
I think we should try to find a generic one and we should not split for dots, cause querystring and forms allow the use of dot in their names (even if using dots in the name, it will make things difficult, in js and css in particular).
